internal/pkg/agent/cmd/upgrade.go

```diff
@@ -162,6 +167,15 @@ func upgradeCmdWithClient(input *upgradeInput) error {
 		return fmt.Errorf("failed to retrieve command flag information while trying to upgrade the agent: %w", err)
 	}
 
+	rollback, err := cmd.Flags().GetBool(flagRollback)
```

Comment:
`rollback` here shadows the `func rollback` maybe change to rollbackFlag?!

Reply:
addressed in 1a376f15f7e145c2c6f73f68580a7de113e9ad99